src/components/Upload.vue

```diff
@@ -74,6 +74,10 @@ export default {
       default: null,
       type: String,
     },
+    uploadPhrase: {
```

Comment:
I don't have strong feelings, but maybe something like "startButtonText" would be more descriptive?

Reply:
That works for me.

Reply:
+1 That also seems to be the format vuetify uses for internationalization phrases - https://vuetifyjs.com/en/customization/internationalization.